iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Substation.java

```diff
@@ -23,6 +23,8 @@
  */
 public interface Substation extends Container<Substation> {
 
+    String GEOGRAPHICAL_TAGS_KEY = "geographicalTags";
```

Comment:
Not sure it should be a field here... Why do you need it as a constant if the point is to delete/deprecate explicit reference to geographicalTags?